partition_table_test.go

```diff
@@ -372,9 +373,9 @@ func TestPT_load(t *testing.T) {
 			consumer               = defaultSaramaAutoConsumerMock(t)
 			topic                  = "some-topic"
 			partition        int32
-			count            int32
+			count                           = new(int64)
```

Comment:
that looks weird, can't w leave it at int64 and call
`atomic.AddInt64(&count, 1)` instead?
I mean this is technically also correct, but it's kind of off-schema.
Someone correct me if this has become standard-style :)